backend/common_python/server_common.py

```
@@ -213,17 +255,112 @@ def register(config: Dict[str, Any], endpoint: str) -> None:
+
     msgpack_encoder = msgspec.msgpack.Encoder()
+    try:
+        while True:
+            client_identity, corr_id_bytes, handler_id_bytes, resps = (
+                response_queue.get()
+            )
+            response_msg = [client_identity, corr_id_bytes, handler_id_bytes] + [
+                msgpack_encoder.encode(r) for r in resps
+            ]
+            socket.send_multipart(response_msg)
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)